sw/demo/main.c

```diff
@@ -33,6 +33,10 @@ int main(void) {
       puts("Hello World! ");
       puthex(last_elapsed_time);
       putchar('\n');
+      puts("Input Value: ");
+      puthex(get_inputs(GPIO0 + GPIO_IN_REG));
```

Comment:
I think I would add a define of GPIO1, I always try to avoid adding constants to pointer values.